engine/src/test/java/com/datatorrent/stram/engine/AutoMetricTest.java

```diff
@@ -361,6 +384,15 @@ public int getMyMetric()
     }
   }
 
+  public static class OperatorAndAggregator extends OperatorWithMetrics implements AutoMetric.Aggregator
+  {
+    @Override
+    public Map<String, Object> aggregate(long windowId, Collection<AutoMetric.PhysicalMetricsContext> physicalMetrics)
+    {
+      return null;
```

Comment:
I think this test aggregator method should actually do something, then verify it in the testDefaultMetricsAggregator method.